Drv/Ip/SocketReadTask.hpp

```diff
@@ -171,7 +171,7 @@ class SocketReadTask {
     Os::Task m_task;
     bool m_reconnect; //!< Force reconnection
     bool m_stop; //!< Stops the task when set to true
-
+    Os::Mutex m_task_lock;
```

Comment:
@JohanBertrand @csmith608 These likely need to be discarded as they affect locking.